app/views/privacy-statements/fr-lang-privacy-statement.ejs

```diff
@@ -0,0 +1,75 @@
+<!-- prettier spacer -->
+
+<%- include('partials/header') -%>
+
+<h1><%= __("Politique de  confidentialité") %></h1>
+<p class="subtitle">What data do we store about you and why?</p>
+
+<h3>Genre</h3>
+<p>
+    Nous enregistrons le genre afin d'étudier la
+    répartition entre les hommes et les femmes présentant des symptômes.
+</p>
+
+<h3>Tranche d'âge</h3>
+<p>
+    Nous enregistrons l'âge pour distinguer les tranches d'âge
+    qui présentent des symptômes.
+</p>
+
+<h3>Code postal</h3>
+<p>
+    Nous enregistrons les codes postaux afin d'avoir un aperçu du nombre de
+    personnes qui présentent des symptômes dans une région donnée.
+    Nous pouvons ainsi étudier si certains symptômes sont spécifiques à une zone
+    géographique.
+</p>
+
+<h3>Résultat du test</h3>
+<p>
+  Nous enregistrons également si vous avez été testé et quels ont été les résultats.
+   Grâce à ces informations, nous pouvons estimer la proportion de participants
+  qui présentent des symptômes. Nous affichons cela sur une carte.
+</p>
+
+<h3>Symptômes</h3>
+<p>
+  Les symptômes que vous rapportez indiquent si vous pouvez être infecté par le Covid-19.
+  La date d'apparition des symptômes donne également une indication de la durée depuis laquelle
+ vous êtes potentiellement contagieux.
+  Nous enregistrons ces données et les analysons de façon anonyme.
+  Ainsi, le public peut avoir une idée approximative du nombre de cas réels.
+</p>
+<div class="border-b border-cvs-red mt-4"></div>
+<h3>Cookies</h3>
+<p>
+    Nous utilisons des cookies pour sauvegarder les données des participants dans leur propre navigateur.
+    Pour cela, nous demandons l'autorisation au participant.
+    Grâce à ces cookies, les participants peuvent mettre à jour leur état de santé quand celui-ci évolue.
+</p>
+
+<h3>Supression des données</h3>
+<p>
+    Toutes les données enregistrées sont anonymisées et nous ne stockons aucune donnée personnelle pouvant
+    être supprimée.
+</p>
+
+<h3>Partage des données avec le service national de santé</h3>
+<p>Si vous le souhaitez, nous fournissons des données anonymes au service national de santé.</p>
+
+<h3>Analytiques</h3>
+<p>
+
+    Nous utilisons des analytiques pour avoir un aperçu de l'utilisation du site Web. Nous utilisons
+  <a class="text-cvs-red hover:underline" href="https://simpleanalytics.com" target="_blank">Simple Analytics</a>
+  pour ça - un outil qui ne stocke pas de données personnelles et ne place pas de
+  cookies.
+</p>
+
+<h3>D'autres questions?</h3>
+<p>
+  Contactez nous sur Twitter:
+  <a class="text-cvs-red hover:underline" href="https://twitter.com/coronastatusNL" target="_blank"
```

Comment:
Should probably create a separate twitter account for France (or just change the line to an email address or just delete it) :)

Reply:
Just created an account : https://twitter.com/CoronastatusFR